doc/Learn/Authentication/HowTo-MsalAuthentication.md

```diff
@@ -48,26 +48,29 @@ uid: Uno.Extensions.Authentication.HowToMsalAuthentication
     }
     ```
 
-- Add the `MsalAuthenticationProvider` using the `AddMsal()` extension method which configures the `IAuthenticationBuilder` to use it.
+- Use the `Configure` method overload that provides access to a `Window` instance. Add the `MsalAuthenticationProvider` using the `AddMsal()` extension method which configures the `IAuthenticationBuilder` to use it.
 
     ```csharp
     private IHost Host { get; set; }
 
     protected override void OnLaunched(LaunchActivatedEventArgs args)
     {
         var builder = this.CreateBuilder(args)
-            .Configure(host =>
+            .Configure((host, window) =>
             {
                 host
                 .UseAuthentication(builder =>
                 {
-                    builder.AddMsal();
+                    builder.AddMsal(window);
                 });
             });
         ...
     }
     ```
 
+> [!IMPORTANT]
+> The `AddMsal()` method requires a `Window` instance, which the `MsalAuthenticationProvider` uses to set up the authentication dialog. You can access the `Window` instance through the `Configure()` method overload that provides it.
```

Comment:
I don't think it'd be required, its only there so the WAM will display properly. Omitting it would fall back to the deprecated method but it wouldnt cause an error